requirements.txt

```diff
@@ -1,2 +1,5 @@
+jsontableschema-sql
+datapackage
+sqlalchemy
```

Comment:
README says that to test you should install from tests/requirements.txt, which is missing sqlalchaemy.